DataManagementSystem/Client/FTS3Job.py

```diff
@@ -200,11 +191,22 @@ def _constructTransferJob( self, context, pinTime, allTargetSURLs, failedLFNs, t
     source_spacetoken = res['Value']
 
     # getting all the source surls
-    res = StorageElement( self.sourceSE ).getURL( allTargetSURLs, protocol = 'srm' )
+    voName = None
```

Comment:
So you removed the voName from some places, but not everywhere. Can you explain me again why do you need it here ?

Reply:
My question is also valid for all the other occurrences